app/Http/Controllers/Controller.php

```diff
@@ -36,4 +36,16 @@ protected function buildFailedValidationResponse(Request $request, $errors)
             'errors' => $data
         ], 422);
     }
+
+    /**
+     * Returns a json success response.
+     *
+     * @return JsonResponse
+     */
+    protected function successResponse()
+    {
+        return new JsonResponse([
+            'status' => 'ok'
+        ]);
```

Comment:
Nice job. Before we we're returning just a string "ok". Please keep using JSON responses as these are the proper convention.